.github/CODEOWNERS

```diff
@@ -254,6 +254,7 @@ packages/react-components/react-list-preview @microsoft/teams-prg
 packages/react-components/react-motions-preview @microsoft/teams-prg
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30226&commitId=d17390b166c07491fb6073041a77ee49f1ae49cf&env=prod&clientType=FLUENTUI)


  <details><summary><b>InfoLabel</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|InfoLabel.default - Dark Mode.rest.chromium.png|39| Changed|
|InfoLabel.default - High Contrast.rest.chromium.png|57| Changed|
|InfoLabel.default.rest.chromium.png|43| Changed|
|InfoLabel.wrap.rest.chromium.png|43| Changed|
|InfoLabel.required.rest.chromium.png|43| Changed|
|InfoLabel.default - RTL.rest.chromium.png|43| Changed|
|InfoLabel.required - RTL.rest.chromium.png|43| Changed|
|InfoLabel.sizes -- open.rest.chromium.png|25| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>